src/task/task.service.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  BadRequestException,
+  HttpCode,
+  HttpStatus,
+  Injectable,
+} from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Task } from 'src/entities/task.entity';
+import { formatDate } from 'src/helpers/date-and-time.helper';
+import { Repository } from 'typeorm';
+import { UpdateTaskDto } from './dto/update-task.dto';
+import { GetUser } from 'src/decorator/get-user.decorator';
+import { User } from 'src/entities/user.entity';
+
+@Injectable()
+export class TaskService {
+  constructor(
+    @InjectRepository(Task)
+    private taskRepository: Repository<Task>,
+  ) {}
+
+  async getDefaultTaskList(
+    workspaceId: number,
+    userId: number,
+    isForCurrentUserOnly = false,
+  ): Promise<{ tasks: Task[]; count: number }> {
+    const qb = this.taskRepository
+      .createQueryBuilder('tasks')
+      .leftJoinAndSelect('tasks.user', 'user')
+      .select('*')
+      .where('tasks.createdAt like :currentDate', {
```

Comment:
No action needed here: This will be changed according to the requirements from the previous class where we said that by default all the tasks which don't have summary id will be shown. We will align this a bit later